packages/web-api-send-notification-job-manager/src/task/send-notification-task-property-provider.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BatchServiceModels } from '@azure/batch';
+import { BatchTaskPropertyProvider } from 'azure-services';
+import { EnvironmentSettings, ServiceConfiguration } from 'common';
+import { inject, injectable } from 'inversify';
+
+@injectable()
+export class SendNotificationTaskPropertyProvider extends BatchTaskPropertyProvider {
+    constructor(
+        @inject(EnvironmentSettings) private readonly environmentSettings: EnvironmentSettings,
+        @inject(ServiceConfiguration) serviceConfig: ServiceConfiguration,
+    ) {
+        super(serviceConfig, 'start-web-api-send-notification-runner.sh', ['id', 'url']);
```

Comment:
we need to change this properties based on what information we have in the queue. will update this list in the next PR